cmd/sandbox/main.go

```diff
@@ -30,12 +30,12 @@ import (
 
 var (
 	policyFile string
-	noNewPrivs bool
+	allowNewPrivs bool
 )
 
 func main() {
 	flag.StringVar(&policyFile, "policy", "seccomp.yml", "seccomp policy file")
-	flag.BoolVar(&noNewPrivs, "no-new-privs", true, "set no new privs bit")
```

Comment:
Couldn't we use `-no-new-privs=false` rather than changing it?

Reply:
aha, I didn't know bool flags could take a value!